stl/src/vector_algorithms.cpp

```diff
@@ -800,7 +800,7 @@ namespace {
 #ifndef _M_ARM64EC
         static constexpr bool _Has_portion_max = false;
 
-        static __m128i _Sign_correction(const __m128i _Val, const bool _Sign) {
+        static __m128i _Sign_correction(const __m128i _Val, const bool _Sign) noexcept {

```

Comment:
This is outside `extern "C"` as far as I can tell (it ended on line 483). This appears to be a good change for consistency with `_Minmax_traits_1/2/4`, but at a minimum it should be called out as a drive-by change.

Similarly for the following changes in this file (where an `extern "C"` block ends on 1125 immediately before them).

Reply:
Yes, these are functions called from `extern "C"` functions - without wrapping in `catch(...)` - which consequently need to be `noexcept`. Not exactly drive-by, but deserving of calling out in the commit message.